db/index.js

```diff
@@ -10,17 +10,36 @@ let writeFile = util.promisify(fs.writeFile);
 const filePath = path.resolve('db/cameras.json');
 
 async function readCameras() {
-  const json = await readFile(filePath);
+  const json = await readFile(filePath, 'utf8');
   return JSON.parse(json);
 }
 
 async function writeCameras(cameras) {
   const json = JSON.stringify(cameras, null, 2);
   await writeFile(filePath, json);
+
+  const json = JSON.stringify(items, null, 2);
+  return writeFile(dbfilepath, json, 'utf8')
 }
 
 async function createCamera(country) {
   const allCountries = await readCountries();
   await writeCountries(allCountries.concat(country));
+
+  items.push(newItem);
+  return writeitems(items);
 }
 
+asyc function getitembyid(id) {
```

Comment:
i like the way you used aysnc await in your code makes all your functions look simple and easy to understand.

Please be careful as your async is spelled wrong.

Reply:
I dont have that in my code. I think thats an old code you are referring to. I had made changes since then and use getCamerabyId with async spelt correctly.